### PR TITLE
CORTX-29741:Redundant confstore keys removal

### PR DESCRIPTION
**Pull request by lakshita-jain**
CORTX-29741: replace obsolete conf-store keys
keys replaced old to new:
cvg_count to num_cvg
storage_set_count to num_storage_set

Signed-off-by: Lakshita Jain <lakshita.jain@seagate.com>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Comment by vaibhavparatwar** [comment]
@lakshita-jain please add the logs for what unit tests have been done for this PR. specially 3N deployment.

**Comment by lakshita-jain** [comment]
3 Node Deployment successful:
[root@ssc-vm-g2-rhev4-3081 ~]# kubectl get po
NAME                                                 READY   STATUS    RESTARTS   AGE
consul-client-bsnj8                                  1/1     Running   0          4h5m
consul-client-kkdfr                                  1/1     Running   0          4h5m
consul-client-rxbx9                                  1/1     Running   0          4h5m
consul-server-0                                      1/1     Running   0          4h4m
consul-server-1                                      1/1     Running   0          4h5m
consul-server-2                                      1/1     Running   0          4h6m
cortx-control-84944546ff-spdc9                       1/1     Running   0          4h3m
cortx-data-ssc-vm-g2-rhev4-3081-cd675bc4-45hrn       3/3     Running   0          4h1m
cortx-data-ssc-vm-g2-rhev4-3082-756dd6bc59-5vcvw     3/3     Running   0          4h1m
cortx-data-ssc-vm-g2-rhev4-3083-757d684848-kcm9r     3/3     Running   0          4h1m
cortx-ha-55b6bd789c-r6wfb                            3/3     Running   0          3h58m
cortx-server-ssc-vm-g2-rhev4-3081-75c7586f44-mpb62   2/2     Running   0          4h
cortx-server-ssc-vm-g2-rhev4-3082-ddbf4fb88-t6pm4    2/2     Running   0          4h
cortx-server-ssc-vm-g2-rhev4-3083-5b4b5c5fb8-l4sfq   2/2     Running   0          4h
kafka-0                                              1/1     Running   0          4h5m
kafka-1                                              1/1     Running   0          4h5m
kafka-2                                              1/1     Running   0          4h5m
zookeeper-0                                          1/1     Running   0          4h6m
zookeeper-1                                          1/1     Running   0          4h6m
zookeeper-2                                          1/1     Running   0          4h6m

**Comment by vaibhavparatwar** [comment]
@mssawant @SwapnilGaonkar7  please review

**Review by vaibhavparatwar — APPROVED** [review]

**Comment by lakshita-jain** [comment]
[root@cortx-data-headless-svc-ssc-vm-g2-rhev4-3081 /]# hctl status
Bytecount:
    critical : 0
    damaged : 0
    degraded : 0
    healthy : 0
Data pool:
    # fid name
    0x6f00000000000001:0x75 'storage-set-1__sns'
Profile:
    # fid name: pool(s)
    0x7000000000000001:0xa8 'Profile_the_pool': 'storage-set-1__sns' 'storage-set-1__dix' None
Services:
    cortx-data-headless-svc-ssc-vm-g2-rhev4-3081
    [started]  hax                 0x7200000000000001:0x22         inet:tcp:cortx-data-headless-svc-ssc-vm-g2-rhev4-3081@22001
    [started]  ioservice           0x7200000000000001:0x25         inet:tcp:cortx-data-headless-svc-ssc-vm-g2-rhev4-3081@21001
    [started]  confd               0x7200000000000001:0x38         inet:tcp:cortx-data-headless-svc-ssc-vm-g2-rhev4-3081@21002
    cortx-data-headless-svc-ssc-vm-g2-rhev4-3083
    [started]  hax                 0x7200000000000001:0x6          inet:tcp:cortx-data-headless-svc-ssc-vm-g2-rhev4-3083@22001
    [started]  ioservice           0x7200000000000001:0x9          inet:tcp:cortx-data-headless-svc-ssc-vm-g2-rhev4-3083@21001
    [started]  confd               0x7200000000000001:0x1c         inet:tcp:cortx-data-headless-svc-ssc-vm-g2-rhev4-3083@21002
    cortx-data-headless-svc-ssc-vm-g2-rhev4-3082  (RC)
    [started]  hax                 0x7200000000000001:0x3e         inet:tcp:cortx-data-headless-svc-ssc-vm-g2-rhev4-3082@22001
    [started]  ioservice           0x7200000000000001:0x41         inet:tcp:cortx-data-headless-svc-ssc-vm-g2-rhev4-3082@21001
    [started]  confd               0x7200000000000001:0x54         inet:tcp:cortx-data-headless-svc-ssc-vm-g2-rhev4-3082@21002
    cortx-server-headless-svc-ssc-vm-g2-rhev4-3082
    [started]  hax                 0x7200000000000001:0x59         inet:tcp:cortx-server-headless-svc-ssc-vm-g2-rhev4-3082@22001
    [started]  rgw                 0x7200000000000001:0x5c         inet:tcp:cortx-server-headless-svc-ssc-vm-g2-rhev4-3082@21501
    cortx-server-headless-svc-ssc-vm-g2-rhev4-3081
    [started]  hax                 0x7200000000000001:0x61         inet:tcp:cortx-server-headless-svc-ssc-vm-g2-rhev4-3081@22001
    [started]  rgw                 0x7200000000000001:0x64         inet:tcp:cortx-server-headless-svc-ssc-vm-g2-rhev4-3081@21501
    cortx-server-headless-svc-ssc-vm-g2-rhev4-3083
    [started]  hax                 0x7200000000000001:0x69         inet:tcp:cortx-server-headless-svc-ssc-vm-g2-rhev4-3083@22001
    [started]  rgw                 0x7200000000000001:0x6c         inet:tcp:cortx-server-headless-svc-ssc-vm-g2-rhev4-3083@21501

**Review by Shreya-18 — APPROVED** [review]

**Comment by mssawant** [comment]
Thanks for the patch @lakshita-jain, looks good. Kindly reword the commit message as,
```
CORTX-29741: replace obsolete conf-store keys
keys replaced old to new:
cvg_count to num_cvg
storage_set_count to num_storage_set

Signed-off-by: Lakshita Jain <lakshita.jain@seagate.com>
```
cc @vaibhavparatwar

**Review by mssawant — APPROVED** [review]

**Comment by mssawant** [comment]
Also please make sure that pre-merge tests pass.

File: provisioning/miniprov/hare_mp/templates/hare.config.conf.tmpl.1-node [hunk 1]
```diff
@@ -48,7 +48,7 @@
       },
       "storage_set": "TMPL_STORAGE_SET_ID",
       "storage": {
-        "cvg_count": "TMPL_CVG_COUNT",
+        "num_cvg": "TMPL_num_cvg",
```
Comment (SwapnilGaonkar7) [inline provisioning/miniprov/hare_mp/templates/hare.config.conf.tmpl.1-node:51]: It should be 'TMPL_NUM_CVG'

**Comment by SwapnilGaonkar7** [comment]
@lakshita-jain Rebase with latest main

**Comment by vaibhavparatwar** [comment]
Not sure why premerge-test and 1N deployment prechecks are stuck. Can someone take a look? @mssawant @SwapnilGaonkar7 @SunilSavanurSeagate

**Comment by mssawant** [comment]
@lakshita-jain, please squash the commits or fix up the commits.

**Comment by lakshita-jain** [comment]
Multiple commits are not allowed to be merged hence raising a new PR

**Pull request closed without merging**